pkg/restapi/types.go

```diff
@@ -125,10 +125,10 @@ type SessionUpdate struct {
 }
 
 type AgentUpdate struct {
-	Id             string                   `json:"id"`
-	State          string                   `json:"state"`
-	SessionsUpdate map[string]SessionUpdate `json:"sessions"`
-	Gpus           []GpuMetrics             `json:"gpus"`
+	Id      string                   `json:"id"`
+	State   string                   `json:"state"`
+	Updates map[string]SessionUpdate `json:"sessions"`
```

Comment:
nit: its a bit confusing to have this be update but the the json string to be 'sessions'?

Reply:
Yes, the name SessionsUpdate seems semantically incorrect and it was bugging me so I switched it to Updates. Overall, I am not happy with this update structure at all anymore. I think a more generic format would be good but I am not sure what that would look like yet.